fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/QuerySegmentAggregator.java

```diff
@@ -317,6 +322,26 @@ private void processFromClauseForId(StringBuilder fromClause, String target) {
         }
     }
     
+    private void processFromClauseForLastUpdated(StringBuilder fromClause, String target) {
+//        if(!queryParamLastUpdateds.isEmpty()) {
+//            // Start the Drived Table
+//            fromClause.append("( SELECT * FROM ");
+//            fromClause.append(target);
+//            fromClause.append("_RESOURCES IR ");
+//
+//            // Process the Condtional
+//            fromClause.append("WHERE ILR.LOGICAL_ID IN ( ");
+//            //fromClause.append(queryParamId.getValues().stream().map(param -> "?" ).collect(Collectors.joining(", ")));
+//            
+//            // Close out the Derived Tables
+//            fromClause.append(" )) ");
+//        } else {
```

Comment:
why the commented out code?

Reply:
that's the code I'm currently working on.  it'll be uncommented in my next series of commits.